docker/compose/docker-compose.sqlite-tika.arm.yml

```diff
@@ -0,0 +1,84 @@
+# docker-compose file for running paperless from the docker container registry.
+# This file contains everything paperless needs to run.
+# Paperless supports amd64, arm and arm64 hardware.
+#
+# All compose files of paperless configure paperless in the following way:
+#
+# - Paperless is (re)started on system boot, if it was running before shutdown.
+# - Docker volumes for storing data are managed by Docker.
+# - Folders for importing and exporting files are created in the same directory
+#   as this file and mounted to the correct folders inside the container.
+# - Paperless listens on port 8000.
+#
+# SQLite is used as the database. The SQLite file is stored in the data volume.
+#
+# iwishiwasaneagle/apache-tika-arm docker image is used to enable arm64 arch
+# which apache/tika does not currently support.
+#
+# In addition to that, this docker-compose file adds the following optional
+# configurations:
+#
+# - Apache Tika and Gotenberg servers are started with paperless and paperless
+#   is configured to use these services. These provide support for consuming
+#   Office documents (Word, Excel, Power Point and their LibreOffice counter-
+#   parts.
+#
+# To install and update paperless with this file, do the following:
+#
+# - Copy this file as 'docker-compose.yml' and the files 'docker-compose.env'
+#   and '.env' into a folder.
+# - Run 'docker-compose pull'.
+# - Run 'docker-compose run --rm webserver createsuperuser' to create a user.
+# - Run 'docker-compose up -d'.
+#
+# For more extensive installation and update instructions, refer to the
+# documentation.
+
+version: "3.4"
+services:
+  broker:
+    image: redis:6.0
+    restart: unless-stopped
+    volumes:
+      - redisdata:/data
+
+  webserver:
+    image: ghcr.io/paperless-ngx/paperless-ngx:latest
+    restart: unless-stopped
+    depends_on:
+      - broker
+      - gotenberg
+      - tika
+    ports:
+      - 8000:8000
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8000"]
+      interval: 30s
+      timeout: 10s
+      retries: 5
+    volumes:
+      - data:/usr/src/paperless/data
+      - media:/usr/src/paperless/media
+      - ./export:/usr/src/paperless/export
+      - ./consume:/usr/src/paperless/consume
+    env_file: docker-compose.env
+    environment:
+      PAPERLESS_REDIS: redis://broker:6379
+      PAPERLESS_TIKA_ENABLED: 1
+      PAPERLESS_TIKA_GOTENBERG_ENDPOINT: http://gotenberg:3000
+      PAPERLESS_TIKA_ENDPOINT: http://tika:9998
+
+  gotenberg:
+    image: thecodingmachine/gotenberg
+    restart: unless-stopped
+    environment:
+      DISABLE_GOOGLE_CHROME: 1
+
+  tika:
+    image: iwishiwasaneagle/apache-tika-arm@sha256:c1b16bcadcb97a08d47a09fc99bea4d47b63cec06bbf4f437b99c2c3b17181eb
```

Comment:
This image is not woking for me.

I get the error

```
[2022-03-22 22:13:31,320] [INFO] [paperless.parsing.tika] Sending /usr/src/paperless/src/../consume/geburtstag.docx to Tika server
[2022-03-22 22:13:49,959] [DEBUG] [paperless.parsing.tika] Deleting directory /tmp/paperless/paperless-dfrfymi6
[2022-03-22 22:13:49,992] [ERROR] [paperless.consumer] Error while consuming document geburtstag.docx: Could not parse /usr/src/paperless/src/../consume/geburtstag.docx with tika server at http://tika:9998: HTTPConnectionPool(host='tika', port=9998): Max retries exceeded with url: /rmeta/text (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xb1ef68b0>: Failed to establish a new connection: [Errno 113] No route to host'))
```

Checking with `docker logs paperless_tika_1` returns:
```
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
standard_init_linux.go:219: exec user process caused: exec format error
```

If I use `image: abhilesh7/apache-tika-arm` it works.

Reply:
The difference is the abhilesh7 image also includes an `armhf` build, while this PR is only `arm64`

Reply:
Ok for clarification I use Raspbian Buster on a Pi 4, which still is 32bit?

Reply:
Yes rpi still defaults to 32, I think. 

IMO this PR should include support for `armhf` / `armv7` (I don't personally understand the difference there), especially since the compose is called `.arm.yml`.

Another idea, possibly overkill, @paperless-ngx/ci-cd - would it make sense to publish our own tika container covering all 3 archs? This way we simply point to our GHCR manifest instead of creating a separate `.arm.yml`. This puts the publishing burden on us but makes setup easier for users.

Reply:
I think that's a good idea.

Reply:
I'm happy to transfer ownership of https://github.com/iwishiwasaneagle/tika-docker to paperless-ngx

Reply:
> abhilesh7/apache-tika-arm

be aware that this images uses tika v1.27 https://github.com/abhilesh/apache-tika-arm/blob/e069f505d84f3fca691ccb081bd22af94af540f1/Dockerfile-full#L25

Reply:
@qcasey @s-martin I did a bit of work to also add armv7 to the Tika 2.1.0 and 1.27 images. This process is automated still. It seems that I didn't do this in the past because docker buildx arm/v7 struggles to use wget with https? weird bug but I've found some work-arounds now.

![image](https://user-images.githubusercontent.com/12383029/159682091-e46a8859-4025-4688-b58b-a3498e64e443.png)

I have tried the new image on my RPi4 64bit and it's working as expected, so could you verify it also works on 32bit with armv7 please.